src/lib/helpers/ArrowHelper.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { ArrowHelper, Color, ColorRepresentation, Vector3 } from 'three'
```

Comment:
ArrowHelper must be imported as `import { ArrowHelper as ThreeArrowHelper } from 'three'` and used as `new ThreeArrowHelper` otherwise compiling the types of this component fails as the components name ("ArrowHelper") and the imports name from `three` ("ArrowHelper") are the same. This applies to all other components.

Reply:
Imports that are only used as a type must be imported with `import type { ColorRepresentation } from 'three'`. Generally these errors are also visible when you run `npm run check`.